modules/apps/site/site-memberships-web/src/main/resources/META-INF/resources/js/actions.js

```diff
@@ -28,12 +29,21 @@ export const ACTIONS = {
 					return;
 				}
 
-				const input = document.createElement('input');
+				const allItems = document.createElement('input');
 
-				input.name = `${portletNamespace}rowIds`;
-				input.value = selectedItems.map((item) => item.value);
+				allItems.name = `${portletNamespace}availableRowIds`;
+				allItems.value = items.map((item) => item.value);
 
-				editUserGroupRoleFm.appendChild(input);
+				editUserGroupRoleFm.appendChild(allItems);
+
+				const selectedItems = document.createElement('input');
```

Comment:
Same as above, a created `input` element shouldn't be stored in a variable named `selectedItems`

Reply:
... and `selectedItems` refers to the selected checkbox items.

Reply:
I think that the confusion comes from the purpose of the variable, compared to its initial value.

You're assigning it a value of an array of checked select inputs later in the code.

And then you're appending that newly created input element as a child of  `editUserGroupRoleFm`.

Disregarding the context of what this or that method returns, it doesn't make sense to append an element named `selectedItems`. It doesn't sound like an element, it sounds like an array of some items.

I'm just telling you this because Brian returns PRs if he find wrong names.

Reply:
That makes sense; I'll change it back to `input`.  Do the `input` constants need to have different names, or can I just re-instantiate it with the same name?

Reply:
If it's in the same scope then it can't/shouldn't be re-instantiated, and as far as naming goes, as specific as possible is best. You can name them by their expected use, or just use something to add context to the `input` you're creating at that time.

Reply:
Thanks @kresimir-coko, I've sent a rebased PR here with the additional commit.  https://github.com/liferay-frontend/liferay-portal/pull/1797